src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -187,19 +192,17 @@ public static function getOrderSystemStatus(): array
 
         $orderSystemStatus['ids_with_failed_tasks'] = $failedOrderIdsWithoutCancelled;
 
-        $orderStatuses = wc_get_order_statuses();
-        unset($orderStatuses['wc-cancelled']);
         $orderQuery = new \WC_Order_Query([
             'meta_key' => OrderHandler::TO_BE_SYNCHRONIZED_META_KEY,
             'meta_value' => 'yes',
             'meta_compare' => '=',
             'orderby' => 'date_created',
             'order' => 'ASC',
+            'status', array_keys($orderStatuses),
```

Comment:
```suggestion
            'status' => array_keys($orderStatuses),
```

Reply:
https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/124#discussion_r1405399165

Reply:
hmm can you remove the status filter line than? i'll approve after

Reply:
Removed.